webapp/app/actions.js

```diff
@@ -1,13 +1,17 @@
 import axios from 'axios'
 
-import * as User from '@core/user/user'
 import * as i18nFactory from '@core/i18n/i18nFactory'
 
-import { LoaderActions } from '@webapp/store/ui'
 import { SystemStatusState } from '@webapp/store/system'
 
 export const appPropsChange = 'app/props/change'
-export const appUserLogout = 'app/user/logout'
+
+const _fetchUserAndSurvey = async () => {
+  const {
+    data: { user, survey },
+  } = await axios.get('/auth/user')
+  return { user, survey }
+}
 
 // ====== INIT
 
```

Comment:
this is going to be refactored to `initSystem` dispatching an action `SYSTEM_INIT = 'system/init'`